### ISSUE TITLE
Righteous orb drop rates do not match classic

### ISSUE THREAD

**Issue by baranzadeoglu** [chromiecraft/chromiecraft#1856]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour

Righteous orb drop rates do not match classic

![image](https://user-images.githubusercontent.com/84640316/134768850-36330a18-ec26-40ee-8522-a7c8350dcc0e.png)

### Expected Blizzlike Behaviour

Lower drop rates, especially on the Archivist Galford.

### Source

https://tbc.wowhead.com/item=12811/righteous-orb

### Steps to reproduce the problem

select creature_template.name, creature_loot_template.entry, creature_loot_template.chance from creature_loot_template join creature_template on creature_loot_template.Entry=creature_template.lootid where creature_loot_template.item="12811";

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/7e2e6f8ee85f82f57a3c6e0f6f645b4c42292e7a

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by phonkala** [comment]
This isn't classic, it's WotLK with progressive level caps/brackets. Drop rates match WotLK values https://wotlkdb.com/?item=12811

**Comment by baranzadeoglu** [comment]
> This isn't classic, it's WotLK with progressive level caps/brackets. Drop rates match WotLK values https://wotlkdb.com/?item=12811

that is unreliable.

**Comment by phonkala** [comment]
> > This isn't classic, it's WotLK with progressive level caps/brackets. Drop rates match WotLK values https://wotlkdb.com/?item=12811
> 
> that source is unreliable. See this issue: #1741

There is no reliable source before Blizzard releases WotLK Classic server and even that might have them modified from original. We do know drop rates were changed in WotLK so all you can do is to guess, and definitely not use same values than in (vanilla) Classic or TBC Classic.

**Comment by baranzadeoglu** [comment]
> > > This isn't classic, it's WotLK with progressive level caps/brackets. Drop rates match WotLK values https://wotlkdb.com/?item=12811
> > 
> > 
> > that source is unreliable. See this issue: #1741
> 
> There is no reliable source before Blizzard releases WotLK Classic server and even that might have them modified from original. We do know drop rates were changed in WotLK so all you can do is to guess, and definitely not use same values than in (vanilla) Classic or TBC Classic.

did you even read the discussion on #1741 ?

**Comment by phonkala** [comment]
> > > > This isn't classic, it's WotLK with progressive level caps/brackets. Drop rates match WotLK values https://wotlkdb.com/?item=12811
> > > 
> > > 
> > > that source is unreliable. See this issue: #1741
> > 
> > 
> > There is no reliable source before Blizzard releases WotLK Classic server and even that might have them modified from original. We do know drop rates were changed in WotLK so all you can do is to guess, and definitely not use same values than in (vanilla) Classic or TBC Classic.
> 
> did you even read the discussion on #1741 ?

Yes. As said, there are no official sources for WotLK data. If you provide such, of course values for drop rates probably are quickly changed. Until that best guess is that private server db data is kinda close to correct as we know tbc or vanilla data sure isn't same as is in WotLK. Most likely many different WotLK pserver cores got the values closer correct than tbc or vanilla data.

Unsubscribing as there is no point in this entry, the values are what they are until there is some official reference or some pserver decides to change them based on anything they like.
